tensorflow/python/kernel_tests/sparse_ops_test.py

```diff
@@ -743,5 +743,46 @@ def testMismatchedShapes(self):
         tf.sparse_maximum(sp_zero, sp_one).eval()
 
 
+class SparseTransposeTest(tf.test.TestCase):
```

Comment:
Okay sure, just wanted to have a basic test to see if I was doing it correct. Will move the tests to the ``transpose_op_test.py``.